app/src/main/java/com/github/h3lp3rs/h3lp/messaging/RecentMessagesActivity.kt

```diff
@@ -65,8 +66,11 @@ class RecentMessagesActivity : AppCompatActivity() {
         fun onChildAdded(conversationId : String) {
             val helper = HelperConversation(conversationId)
             adapter.add(helper)
-            idToItem[conversationId] = helper
 
+            // Create the key pair used to encrypt the conversation from end-to-end
+            createAndSendKeyPair(conversationId, HELPEE)
```

Comment:
This is what I was talking about earlier, It would be better to have this in the await help activity, that way the second user can still message this user even if he hasn't opened the recent messages activity yet.

Reply:
That would be ideal but it's not trivial since we currently only retrieve conversation Ids in the recentMessages, but it's definitely something to look into